src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -55,7 +55,9 @@ const transformJson = async json => {
 };
 
 export default async ({ path: pathname, service, variant }) => {
-  const { json, ...rest } = await fetchPageData(pathname);
+  const { json, ...rest } = await fetchPageData(pathname, {
+    maximumAttempts: 3,
```

Comment:
maybe start off with 2 attempts, just decreases the effect of increased traffic if Ares were down as below